README.md

```diff
@@ -479,52 +479,55 @@ performance.
 
 ##### Decoupled Responses
 
-It is also possible for a backend to send multiple responses for a
-request or not send any responses for a request. A backend may also
+It is also possible for a backend to send one or more responses
+for a request. A backend may also
 send responses out-of-order relative to the order that the request
 batches are executed. Such backends are called *decoupled* backends.
-The decoupled backends use one `ResponseFactory` object per request to keep
-creating and sending any number of responses for the request. For this
+
+The decoupled backends use one `ResponseFactory` object per request to
+create and send any number of responses for the request. They must send at
+least one final response, even if it is empty and the client opts out of
```

Comment:
```suggestion
least one final response per request (even if it contains no output tensors).
```

Reply:
I think it is best to leave out mentioning the client here - again 'client' can be c api, grpc, http, etc. 

so the backend should just consider it a request - and be agnostic to how it got there -